source/common/grpc/codec.h

```diff
@@ -134,6 +139,9 @@ class Decoder : public FrameInspector {
   // Indicates whether it has buffered any partial data.
   bool hasBufferedData() const { return state_ != State::FhFlag; }
 
+  // Configures the maximum frame length.
+  void setMaxFrameLength(uint32_t max_frame_length) { max_frame_length_ = max_frame_length; }
```

Comment:
Yes, it will be added in next PR. codec client, for example, gRPC async client will configure this option.
Currently in this PR, `setMaxFrameLength ` is configured and tested in codec_test.cc